cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -47,87 +50,134 @@ <h1>{{ page.title }}</h1>
-            opportunity employer and seeks to create and maintain a vibrant
-            and diverse workforce. Women, minorities, veterans, and
-            people with disabilities are encouraged to apply.
-        </em>
-    </section>
+                    {% if page.additional_section_title and page.additional_section_content %}
+                        <div class="block block__sub">
+                            <h4>{{ page.additional_section_title }}</h4>
+                            {{ parse_links( page.additional_section_content ) | safe }}
+                        </div>
+                    {% endif %}
 
+                    {% if page.service_type and page.job_length %}
+                        <div class="block block__sub">
+                            <h4>Position type</h4>
+                            {{- page.service_type }}, {{ page.job_length | lower -}}
+                        </div>
+                    {% endif %}
+                </div>
+            </div>
+            <a class="o-expandable_target o-expandable_link a-link" title="Expand content">
+                <span class="a-link_text">
+                    <span class="o-expandable_cue o-expandable_cue-open">
+                        Read full description
+                        <span class="cf-icon cf-icon-plus-round"></span>
+                   </span>
+                    <span class="o-expandable_cue o-expandable_cue-close">
+                        Hide full description
+                        <span class="cf-icon cf-icon-minus-round"></span>
+                   </span>
+                </span>
+            </a>
+        </div>
 
+    </section>
+    
     <section class="block
                     block__padded-top
-                    block__border-top">
-        <h2 id="interested">Interested in applying?</h2>
+                    block__border-top
+                    vacancy-announcement_apply">
+        <h2 id="interested">What to know if you apply</h2>
         <div class="block block__sub">
-            <h3>Before you apply</h3>
-            <ul class="m-list m-list__links">
-                <li class="m-list_item">
-                    <a class="a-link a-link__jump"
-                       href="/about-us/careers/working-at-cfpb/">
-                        Learn about working @ CFPB
-                    </a>
-                </li>
-                <li class="m-list_item">
-                    <a class="a-link a-link__jump"
-                       href="/about-us/careers/application-process/">
-                        Learn about the application process
-                    </a>
-                </li>
-                <li class="m-list_item">
+            <div class="content-l">
+                <div class="content-l_col content-l_col-1-2">
+                    <div class="embed-container">
+                        <iframe src="https://www.youtube.com/embed/XCbZnTIeTOY" frameborder="0" allowfullscreen></iframe>
```

Comment:
Can we use our internal video player instead of an iframe here?

Reply:
We tried both options, and the youtube player seemed to work better here visually given the small size.

Reply:
Ah ok. Would it be worth adding a comment here with some background so no one tries to swap it out in the future?